diff -up breeze-gtk-6.0.1/CMakeLists.txt.omv~ breeze-gtk-6.0.1/CMakeLists.txt
--- breeze-gtk-6.0.1/CMakeLists.txt.omv~	2024-03-06 08:12:04.306978150 +0100
+++ breeze-gtk-6.0.1/CMakeLists.txt	2024-03-06 08:12:10.707045199 +0100
@@ -4,7 +4,7 @@ project(breeze-gtk)
 set(PROJECT_VERSION "6.0.1")
 set(PROJECT_VERSION_MAJOR 6)
 
-find_package(ECM 6.0.1 REQUIRED NO_MODULE)
+find_package(ECM 6.0.0 REQUIRED NO_MODULE)
 include(FeatureSummary)
 
 set(CMAKE_MODULE_PATH ${ECM_MODULE_PATH}  "${CMAKE_SOURCE_DIR}/cmake")
